src/OnlineRetailPortal.Contracts/Contracts/UploadImageRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts.Contracts
```

Comment:
resolve the namespace to ` OnlineRetailPortal.Contracts`

Reply:
Okay, Done